cmd/util/util.go

```diff
@@ -0,0 +1,39 @@
+package util
+
+import (
+  "fmt"
+  "os"
+  "os/exec"
+  "strings"
+
+  "github.com/fatih/color"
+)
+
+func Contains(list []string, item string) bool {
+  for i := 0; i < len(list); i++ {
+    if item == list[i] {
+      return true
+    }
+  }
+  return false
+}
+
+func ExitWithErrorMessage(message string) {
+  errHeaderFmt := color.New(color.Bold).Add(color.FgRed)
+  errMessageFmt := color.New(color.FgRed)
```

Comment:
This API for colors feels pretty terrible, let's use Aurora. I'm happy to make that change.